src/Tribe/Service_Providers/Context.php

```diff
@@ -259,6 +259,15 @@ static function ( $data ) {
 				],
 				'tec_post_type'        => [
 					'read' => [
+						Tribe__Context::FUNC          => [
+							static function () {
+								return count( array_filter( [
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line

<sub>hash: 73d50a3c7e17b3fc464bb5ed7b45af14</sub>